core/collection/timingwheel.go

```diff
@@ -162,6 +162,11 @@ func (tw *TimingWheel) Stop() {
 	close(tw.stopChannel)
 }
 
+func (tw *TimingWheel) HasKey(key any) bool {
```

Comment:
Contains?

Reply:
Well, `Contains(key) bool` is okay, but `HasKey(key) bool` would be more straightforward.

Especially when the argument isn't named as `key`, eg:

```go
var id string

// ...

tw.Contains(id)  // What contains ID? ID is contained as a task? key? value?

// vs

tw.HasKey(id)  // It's clear that timing wheel has ID as a key.
```